components/istio-kyma-patch/patch.sh

```diff
@@ -135,6 +160,7 @@ function check_requirements() {
 require_istio_system
 require_istio_version
 require_mtls_disabled
+configure_policy_checks_setting
```

Comment:
1. After `check_requirements`
2. I think `configure_policy_checks` would be enough